src/Arcus.WebApi.Security.Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -36,7 +68,7 @@ public async Task<bool> IsValidToken(string token)
         {
```

Comment:
Should async methods be suffixed with Async ?  Or don't we do that within arcus ?

Reply:
Yes, but it's enforced by `IJwtTokenReader`

Reply:
True, but the `IJwtTokenReader` is owned by this library as well ?  (It is defined here).

Reply:
Ahaha I'm a moron 😂 Yes, I'll change it